Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -43,7 +43,6 @@
 #import "WebURLSchemeHandlerCocoa.h"
 #import "_WKApplicationManifestInternal.h"
 #import "_WKVisitedLinkStoreInternal.h"
-#import <WebCore/RuntimeApplicationChecks.h>
```

Comment:
Seems weird that we don't need to include a `RuntimeApplicationChecks.h` at all, especially when you call them below.

This probably breaks a non-unified build -- maybe you should just add the `<wtf/cocoa/RuntimeApplicationChecks.h>` file?

Reply:
`RuntimeApplicationChecksCocoa.h` was already imported on line 57, so I'm just removing a duplicate import.